src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -33,7 +33,8 @@ import { useUser } from "hooks/useUser";
 import { useVenueEvents } from "hooks/events";
 
 import { Button } from "components/atoms/Button";
-import { Schedule } from "components/molecules/Schedule";
+// import { Schedule } from "components/molecules/Schedule";
```

Comment:
you can delete this comment

Reply:
fixed in afad3a98d9284b6bcbe5abf52faa0c6d32e961c4